sunpy/instr/goes.py

```diff
@@ -1357,7 +1357,6 @@ def flux_to_flareclass(goesflux: u.watt/u.m**2):
 
 
 def _assert_chrono_order(obstime):
-    chrono_check = np.array(obstime) - np.roll(obstime, 1)
```

Comment:
`np.roll(obstime)` was returning an array of `Time` (instead of just a `Time`), which was causing this to fail.